### DISCUSSION TITLE
Changes in Sysmon Plays

### DISCUSSION THREAD

**Discussion by ben-sec** [Security-Onion-Solutions/securityonion#8105] [Unsupported Versions]
Hello!

I'm not sure if this is the right place for my question, but do you have an idea why "winlog.channel.security:"Microsoft\-Windows\-Sysmon\/Operational"" has been removed from some Sysmon related Plays (e.g. #479 and #489 elastalert configuration)? Especially #489 is now firing Playbook alarms for all kinds of  event_data.event.dataset (not only Sysmon logs, e.g. zeek conn dataset) in SO. I don't really understand why, because the elastalert alarm should be triggered by specific values for the field "process.executable.security". which is not part of event.dataset: "conn". Ideas?

Cheers, Ben

**Comment by defensivedepth** [comment]
Can you list the full `id` for the Sigma rule that the Play is based off (https://github.com/SigmaHQ/sigma/blob/master/rules/windows/image_load/image_load_abusing_azure_browser_sso.yml#L2) or share a screenshot? 

The numbers you listed (479, 489) are specific to your environment, so I am not sure which ones they are.

**Reply by ben-sec** [reply]
#479 equals rule ID b69888d4-380c-45ce-9cf9-d9ce46e67821 (Detects the creation of an ADS data stream that contains an executable (non-empty imphash) and #489 equals rule ID db809f10-56ce-4420-8c86-d6a7d793c79c (Raw disk access using illegitimate tools, possible defence evasion).

**Reply by batref** [reply]
I am seeing the same behavior with at least the following plays:

Play 479 (Executable in ADS):
id: b69888d4-380c-45ce-9cf9-d9ce46e67821
This was previously depending on Sysmon EventID 15.

Play 489 (Raw Disk Access Using Illegitimate Tools):
id: db809f10-56ce-4420-8c86-d6a7d793c79c 
This was previously depending on Sysmon EventID 9.

Play 491 (Suspicious Remote Thread Created):
id: 66d31e5f-52d6-40a4-9615-002d3789a119
This was previously depending on Sysmon EventID 8.

**Reply by defensivedepth** [reply]
Ok so looking at 489 - Here is the rule in the upstream repo: https://raw.githubusercontent.com/SigmaHQ/sigma/master/rules/windows/raw_access_thread/sysmon_raw_disk_access_using_illegitimate_tools.yml

Right off the bat, you can see that the level is set to low because of false positives. Just from that, you might consider just disabling this rule until the upstream source has been able to tune it better.

`level: low  # far too many false positives`

Secondly, I converted it inside Playbook and am coming up with the following elasticsearch query. This is definitely going to alert on a bunch of stuff because it it doesnt have a logsource mapping defined, so it's just saying: `Alert on any log that does not match the following` (which is probably most of the data in your SO instance.) 

I have created and issue to track adding these mappings that will fix it - I would suggest disabling these for now. Also, if you see any other ones causing issues, please post them here and I will add them to the list. https://github.com/Security-Onion-Solutions/securityonion/issues/8117

`(NOT ((Device.security:*floppy*) OR (process.executable.security:(C\:\\Windows\\System32\\* OR C\:\\Windows\\SystemApps\\* OR C\:\\Windows\\WinSxS\\* OR C\:\\Windows\\servicing\\* OR C\:\\Windows\\CCM\\* OR C\:\\Windows\\uus\\* OR C\:\\Windows\\WinSxs\\*)) OR (process.pid.security:"4") OR (process.executable.security:("C\:\\Windows\\UUS\\amd64\\MoUsoCoreWorker.exe" OR "C\:\\Windows\\explorer.exe")) OR (process.executable.security:("System" OR "Registry")) OR (process.executable.security:*\\Keybase\\upd.exe) OR (process.executable.security:C\:\\ProgramData\\Microsoft\\Windows\ Defender\\Platform\\* AND process.executable.security:*\\MsMpEng.exe) OR (process.executable.security:(C\:\\Program\ Files\\* OR C\:\\Program\ Files\ \(x86\)\\*)) OR (process.executable.security:*C\:\\Users\\* AND process.executable.security:*\\AppData\\* AND process.executable.security:*\\Microsoft\\*) OR (process.executable.security:C\:\\Windows\\Temp\\asgard2\-agent\\* AND process.executable.security:*\\thor.exe) OR (process.executable.security:C\:\\Windows\\SystemApps\\Microsoft.Windows.StartMenuExperienceHost* AND process.executable.security:*\\StartMenuExperienceHost.exe)))`

**Reply by ben-sec** [reply]
Hi Josh, thanks a lot. I already disabled them. Cheers, Ben

**Reply by batref** [reply]
Hi Josh, thanks for looking in to this!

You are indeed correct that 489 gives way to many false positives but this is only true, at least for me, in the new version which has had "service: sysmon" removed from the play.

New play with problem:
Play 492 (WMI Event Subscription):
id: 0f06a3a5-6a09-413f-8743-e6cf35561297
category: wmi_event
This was previously depending on Sysmon EventID 19, 20, and 21

If you check the diff on all the mentioned plays so far, you will see that they have all had "service: sysmon" removed, as ben-sec mentioned.
Most of them also changed from "status: experimental" to "status: test".

I will post if I see more of these.
